packages/odyssey-storybook/src/contributing/OdysseyDevelopment.stories.mdx

```diff
@@ -0,0 +1,87 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Contributing/Odyssey Development" />
+
+# Odyssey development
+
+# Repository structure
+
+The repo is managed via [lerna](https://github.com/lerna/lerna) and [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/)
+and applies a modular approach to code organization where individual
+packages are separated for a number of reasons:
+
+- Delineate private vs public code
+- Share configuration and behavior easily
+- Enforce strong domain boundaries across our dependency graph to allow
+  for easier long term maintenance.
+
+## Working locally
+
+Odyssey uses the Gallium LTS Node version. Ensure you have the correct
+version of `node` installed as well as `yarn` available on your `$PATH`.
+You may need to configure your editor, terminal tab, or tmux pane to
+the correct version each time:
```

Comment:
Are we arbitrarily cropping these docs at 80 characters? That makes it super hard to read with any kind of text wrap or when the code window is shorter than 80 characters.

I'm probably more sensitive to it because I both have columns shorter than 80 characters wide and also have text wrap enabled. I personally prefer responsive text.

I understand this pattern is widespread and wouldn't be able to change; therefore, this is **NON-BLOCKING**.

Reply:
yea I just add a line break myself based on line length, happy to revisit this in another PR though!